sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -43,7 +43,13 @@ export function startRecorder(that: any): Recorder {
       AZURE_TENANT_ID: "azuretenantid"
     },
     customizationsOnRecordings: [],
-    queryParametersToSkip: []
+    queryParametersToSkip: [],
+    requestBodyTransformations: {
+      stringTransforms: [
+        (body: string) =>
+          body.replace(/client-request-id=[^&]*/g, "client-request-id=client-request-id")
```

Comment:
Can we make this a default body transformation?

Reply:
I meant to do it.